sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -384,7 +404,8 @@ private[ui] class ExecutionPagedTable(
 
   private def descriptionCell(execution: SQLExecutionUIData): Seq[Node] = {
     val details = if (execution.details != null && execution.details.nonEmpty) {
-      <span onclick="this.parentNode.querySelector('.stage-details').classList.toggle('collapsed')"
+      <span
+      onclick="this.parentNode.querySelector('.stage-details').classList.toggle('collapsed')"
```

Comment:
Why do you break line 387 into two lines?

Reply:
it's contain 100 character. that's by i have break line into two lines

Reply:
100 is allowed. If `dev/scalastyle` doesn't complain, we should not touch the other lines.

Reply:
ok

Reply:
I think, still it shows the difference.